common_types/src/schema.rs

```diff
@@ -884,34 +884,33 @@ impl Schema {
         }
     }
 
-    pub fn unique_keys(&self) -> Vec<&str> {
-        // Only filters on the columns belonging to the unique key can be pushed down.
-        if self.tsid_column().is_some() {
-            // When tsid exists, that means default primary key (tsid, timestamp) is used.
-            // So, all filters of tag columns(tsid is the hash result of all tags),
-            // timestamp key and tsid can be pushed down.
-            let mut keys = self
-                .columns()
-                .iter()
-                .filter_map(|column| {
-                    if column.is_tag {
-                        Some(column.name.as_str())
-                    } else {
-                        None
-                    }
-                })
-                .collect::<Vec<_>>();
-            keys.extend([&self.tsid_column().unwrap().name, self.timestamp_name()]);
+    pub fn is_unique_column(&self, column: &str) -> bool {
```

Comment:
`col_name` may be a better name here:
```suggestion
    pub fn is_unique_column(&self, col_name: &str) -> bool {
```